example/sample-koin/src/commonMain/kotlin/composegears/tiamat/sample/koin/KoinIntegrationScreen.kt

```diff
@@ -33,15 +35,19 @@ val KoinIntegrationScreen by navDestination<Unit> {
 
 private val KoinListScreen by navDestination<Unit> {
     val navController = navController()
+    val saveableViewModel = koinSaveableTiamatViewModel<SaveableViewModel>()
     val sharedViewModel = koinSharedTiamatViewModel<SharedViewModel>()
 
     SimpleScreen("Koin integration") {
         Column(
             horizontalAlignment = Alignment.CenterHorizontally,
             verticalArrangement = Arrangement.spacedBy(8.dp)
         ) {
-            val timer by sharedViewModel.timer.collectAsState()
-            ViewModelInfo(hashCode = sharedViewModel.hashCode(), timer = timer)
+            val timer1 by sharedViewModel.timer.collectAsState()
+            val timer2 by saveableViewModel.timer.collectAsState()
+            ViewModelInfo("SharedViewModel", hashCode = sharedViewModel.hashCode(), timer = timer1)
```

Comment:
the same and below, can be with named args to make code consistent

Reply:
Below one is not "shared" it is "saveable" - another name